user/views.py

```diff
@@ -69,4 +69,94 @@ def post(self, request):
 
         except KeyError:
             return JsonResponse({'message':'KEYERROR'}, status=400)
-                
\ No newline at end of file
+
+class UserInfoView(View):
+    @Login_required
+    def get(self, request):
+        user = User.objects.get(id=request.user.id)
+
+        return JsonResponse({'name':user.name})
+
+class LikeProductView(View):
+    @Login_required
+    def post(self, request):
+        user_data = json.loads(request.body)
+
+        if not LikeProduct.objects.filter(product_id = user_data['product_id'], user_id = request.user.id).exists():
+            LikeProduct.objects.create(
+                user    = request.user,
+                product = Product.objects.get(id = user_data['product_id'])
+            ).save
+
+            return JsonResponse({'message':'ADD_LIKE_PRODUCT'}, status=200)
+
+        else:
+            delete_product = LikeProduct.objects.get(product_id = user_data['product_id'], user_id = request.user.id)
+            delete_product.delete()
+
+            return JsonResponse({'message':'DELETE_LIKE_PRODUCT'}, status=200)
```

Comment:
```suggestion
        
        delete_product = LikeProduct.objects.get(product_id = user_data['product_id'], user_id = request.user.id)
        delete_product.delete()
        
        return JsonResponse({'message':'DELETE_LIKE_PRODUCT'}, status=200)
```

굳이 `else`가 없어도 if문이 만족되지 않으면 도달할 수 있는 코드입니다.
불필요한 `else`는 사용하지 않는 것이 파이썬다운 코드입니다. :)